### ISSUE TITLE
extend aircraft list

### ISSUE THREAD

**Issue by GliderGeek** [skylines-project/skylines#638]
Ventus 2cxa 18m FES

don't know if there are other aircraft types which are availble on OLC but not on skylines

**Comment by GliderGeek** [comment]
where can i find the planes which are listed in skylines?
also, can we get the OLC list?

**Comment by Turbo87** [comment]
@GliderGeek the plane list is part of the database. We've imported the DMSt index list a few years ago and then tried to maintain it manually. My long-term plan is to remove the plane type list and enable everyone to create their own list of planes.

**Comment by GliderGeek** [comment]
what is the benefit of this approach?

i can imagine that it would be useful to have the same plane formatting (e.g. ventus 2, Ventus 2, Ventus2, ventus2).

or am i misunderstanding your plan for own creation of lists?

**Comment by Turbo87** [comment]
> i can imagine that it would be useful to have the same plane formatting (e.g. ventus 2, Ventus 2, Ventus2, ventus2).

I agree on that and there are at least two ways to address that problem:

- use the current list as a form of "template" that can be used
- suggest existing similar plane type names

the big thing I haven't figured out yet is the ownership model. I don't think it would be useful if every user had to create those plane records for himself because it would make the first time experience a lot more complicated...

**Comment by GliderGeek** [comment]
What problem are we solving with these personalised plane records?
I don't mind adding new types now and then. If we make it easy for users to report missing planes, this should not be a big problem?

**Comment by Turbo87** [comment]
> What problem are we solving with these personalised plane records?

I guess they would enable us to reference them with some other things (e.g. live tracking).

> I don't mind adding new types now and then. If we make it easy for users to report missing planes, this should not be a big problem?

I just thought maybe we should do something similar to https://github.com/XCSoar/xcsoar-data-repository and create some sort of data repository of glider/paraglider types and data that could be imported by SkyLines and others... any thoughts on that?
